packages/mdc-button/mdc-button.scss

```diff
@@ -149,10 +142,6 @@
   &--accent {
     @include mdc-theme-prop(color, accent);
 
-    @include mdc-theme-dark(".mdc-button") {
```

Comment:
Removing this changes the color of the `default with accent` button in dark mode.

### Before (current prod behavior):
![mdc-web-pr-1089-before](https://user-images.githubusercontent.com/409245/29050032-559df650-7b8e-11e7-8d41-f397802f491c.png)

### After (this PR):
![mdc-web-pr-1089-after](https://user-images.githubusercontent.com/409245/29050031-5592f96c-7b8e-11e7-8d71-2328580d61ac.png)